ros/api/v1/hosts.py

```diff
@@ -319,6 +322,8 @@ def get(self, host_id):
             record['os'] = system.deserialize_host_os_data
             record['number_of_recommendations'] = profile.number_of_recommendations
             record['psi_enabled'] = profile.psi_enabled
+            if not get_flag_value(FLAG_INVENTORY_GROUPS):
+                record['groups'] = []
```

Comment:
@upadhyeammit can you please share why we added this code? If I am not wrong this is for performance profile? If yes I am not sure if we need group info there.

Reply:
I think then maybe we don't even need the Groups column for Executive report?

Reply:
@upadhyeammit I don't see any use case ATM where it will be useful in Executive report so I feel we dont need it there. But let me know if you can see any usecase.

Reply:
I removed it from the pprofile.